packages/fcl/src/current-user/normalize/authn-refresh.js

```diff
@@ -0,0 +1,22 @@
+// {
+//   "f_type": "Service",
+//   "f_vsn": "1.0.0",
+//   "type": "authn-refresh",
+//   "uid": "uniqueDedupeKey",
+//   "endpoint": "https://rawr",
+//   "method": "HTTP/POST", // HTTP/POST | IFRAME/RPC | HTTP/RPC
```

Comment:
Is this restricted to being a backchannel service? Or will it support front channel as well?

Reply:
I thiink the idea is backchannel and provide a local view if user interaction is needed